docs/dev/docker-hub.md

```diff
@@ -139,7 +139,7 @@ $ docker run --name some-scylla --volume /var/lib/scylla:/var/lib/scylla -d scyl
 ### Configuring resource limits
 
 The ScyllaDB docker image defaults to running on overprovisioned mode and won't apply any CPU pinning optimizations, which it normally does in non-containerized environments.
-For better performance, it is recommended to configure resource limits for your Docker container using the `--smp`, `--memory`, and `--cpuset` command line options, as well as 
+For better performance, it is recommended to configure resource limits for your Docker container using the `--smp`, `--memory`, and `--cpuset` command line options, as well as
```

Comment:
```suggestion
For better performance, it is recommended to configure resource limits for your container using the `--smp`, `--memory`, and `--cpuset` command line options, as well as
```

It doesn't have to be Docker. Could be Podman.

Reply:
This document is called `docker-hub.md`, and is entirely focused on docker. It mentions docker on 39 lines (and there are lines that reference docker multiple times). AIUI, the scope of this patch is only to update the `aio-max-nr` calculation, which is out of date / incorrect.

(@guy9 is working on #19921; I proposed there that I fix up the calculation in `docs/dev/docker-hub.md` first, then he should please extend `docs/operating-scylla/procedures/tips/best-practices-scylla-on-docker.rst` based on `docker-hub.md`. However... `best-practices-scylla-on-docker.rst` is also entirely focused on docker. De-dockerizing our container documentation would likely need a separate endeavor.)